subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/FileSystemWatchingHelper.java

```diff
@@ -22,7 +22,7 @@
 
 public class FileSystemWatchingHelper {
 
-    private static final int WAIT_FOR_CHANGES_PICKED_UP_MILLIS = 120;
+    private static final int WAIT_FOR_CHANGES_PICKED_UP_MILLIS = 500;
```

Comment:
Maybe this should be dependent on the JDK? Like 500ms for Java 8 given what you've found, but keep 120 for 11+?

Reply:
Good point, I did that!